### PR TITLE
ISSUE: #80 - Reduce d3 bundlesize in client-participation

### PR DESCRIPTION
**Pull request by patcon**
**Addresses issue: #80**

* Replaces global 3rdparty d3 (v4) with individual submodules locked to proper versions:`d3-dsv`, `d3-scale`, `d3-shape`, `d3-force`
* reduces `client-participation/dist/js/participation_bundle.*.js` (gzipped) bundlesize **from 521 KiB to 452 KiB**
  * saving technically come from replacing 3rdparty module with `d3-dsv` in `client-participation/js/stores/polis.js`

Screen Shot 2023-04-06 at 2 27 26 PM

## Checklist

- [x] Title is in format `ISSUE: #XX - Brief description`
- [x] I have updated `CHANGELOG.md`
- [ ] I have run `e2e` tests and fixed any failures ~~(will let CI run)~~
- [ ] I have run `clojure -M:test` and fixed any failures ~~(will let CI run)~~

### CODE REVIEW

**Comment by patcon** [comment]
Ah shoot, I'm not set up to run the whole environment, just ran client-participation locally to confirm the savings in bundlesize. I'll have to get setup for e2e tests later (I might need to run it in the cloud, as my local machine is strapped for resources)

**Comment by patcon** [comment]
Is there a reason the tests were removed from CI btw? Could I offer to add them back? (I often would rely on them, instead of running locally)

**Comment by patcon** [comment]
Is a new "Performance" category in readme appropriate? Wasn't sure if this was bug or feature, but happy to change that

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,4 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#1](https://github.com/DFE-Digital/polis-whitelabel/issues/1) Fixed `authorUid is undefined` crash in `server`
 - [#3](https://github.com/DFE-Digital/polis-whitelabel/issues/3) Fixed crash when `.headersJson` file not present for `file-server`
 
+### Performance
```
Comment (brendanarnold) [inline CHANGELOG.md:39]: Good shout - I've had this problem as well - maybe make it a 'Refactor' title to encompass this and some other similarly not-really-bugs-or-features?

**Comment by brendanarnold** [comment]
Hey @patcon thanks for doing this - re: 

> Ah shoot, I'm not set up to run the whole environment, just ran client-participation locally to confirm the savings in bundlesize. I'll have to get setup for e2e tests later (I might need to run it in the cloud, as my local machine is strapped for resources)

You should be able to setup the enviornment by opening in VSCode with the Devcontainer extension and Docker installed on your local machine - instructions are at https://github.com/DFE-Digital/polis-whitelabel/blob/govuk/main/.devcontainer/README.md Need to `npm install` and run each module but let me know if you run into trouble, obvs it does need a bit of resource for Docker

**Comment by brendanarnold** [comment]
> Is there a reason the tests were removed from CI btw? Could I offer to add them back? (I often would rely on them, instead of running locally)

Is this through the Github CI? So long as they don't cost to run and don't block the merge I'm happy to have them back in.

**Pull request merged**